recipes/pychef/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "PyChef" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "afc2a538fc0492b387f43de72af21ba829cb5b3d2b5e46861a8d15523845bf7d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - requests
+    - setuptools
+    - six
+  run:
+    - python
+    - requests
+    # pychef makes used of pkg_resources
+    - setuptools
+    - six
+
+test:
+  imports:
+    - chef
+
+about:
+  home: https://github.com/coderanger/pychef
+  license: Apache 2.0
+  license_family: Apache
+  # license_file: LICENSE
```

Comment:
Could we please raise an issue or PR upstream? Also please xref it back here when done.